stdlib/REPL/test/docview.jl

```diff
@@ -0,0 +1,13 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+import REPL
+
+@test startswith(let buf = IOBuffer()
+        Core.eval(Main, REPL.helpmode(buf, "α"))
+        String(take!(buf))
+    end, "\"α\" can be typed by \\:alpha:<tab>\n")
```

Comment:
No, there are only colons in the emoji completion (e.g. `:koala:`), not in the Greek letters.

In general, I would suggest trying out your tests locally to cut down on the back-and-forth.